### PR TITLE
add grow/shrink shorter version

### PR DESCRIPTION
**Pull request by lookis**
closes: https://github.com/jaredh159/tailwind-react-native-classnames/issues/139

### CODE REVIEW

File: src/__tests__/flex.spec.ts [hunk 1]
```diff
@@ -12,6 +12,10 @@ describe(`flex grow/shrink`, () => {
     [`flex-grow-0`, { flexGrow: 0 }],
     [`flex-grow`, { flexGrow: 1 }],
     [`flex-grow-77`, { flexGrow: 77 }],
+    [`grow`, { flexGrow: 1 }],
+    [`grow-0`, { flexGrow: 0 }],
+    [`shrink`, { flexShrink: 1 }],
+    [`shrink-0`, { flexShrink: 0}],
```
Comment (jaredh159) [inline src/__tests__/flex.spec.ts:18]: can you add a couple of tests to show that arbitrary grow/shrink values work as well? like `grow-33` and `shrink-77`?

Comment (lookis) [reply src/__tests__/flex.spec.ts]: thanks, added some more test

**Comment by jaredh159** [comment]
thanks again! merging. 👍

**Comment by jaredh159** [comment]
available as `3.0.4` if you want to use the new version -- `npm install twrnc@latest`

**Pull request merged**